### ISSUE TITLE
Pytest-BDD results in Reportportal (Reference https://github.com/reportportal/reportportal/issues/1920)

### ISSUE THREAD

**Issue by globaldeepa1** [reportportal/agent-python-pytest#357]
Hi Team,

We are using Pytest-BDD framework for test automation. We are not able integrate our results into ReportPortal.
Since this tool is developed on top of Pytest, we are able to upload results in Pytest format but that is not including scenario & steps used in BDD format.

Can you please check if any quick solution available around this.

Thanks & Regards,
Deepa

**Comment by HardNorth** [comment]
@globaldeepa1, no, there is no magic bullet here. It requires serious additional implementation.

**Comment by HardNorth** [comment]
@globaldeepa1 Done, please use this release: https://github.com/reportportal/agent-python-pytest/releases/tag/5.5.0
